packages/components/src/internal/components/editable/EditableGridPanel.tsx

```diff
@@ -103,6 +107,7 @@ export const EditableGridPanel: FC<Props> = memo(props => {
     const editorModels = Array.isArray(editorModel) ? editorModel : [editorModel];
     const activeEditorModel = editorModels[activeTab];
     const hasTabs = models.length > 1;
+    let wasDirty = false;
 
     const _onChange = useCallback(
         (editorModelChanges: Partial<EditorModelProps>, dataKeys?: List<any>, data?: Map<any, Map<string, any>>) =>
```

Comment:
Should _onChange call setIsDirty(true)?

Reply:
Leaving that to the `onChange` callback (for now).